### DISCUSSION TITLE
Hacktoberfest2022

### DISCUSSION THREAD

**Discussion by aanya963** [pranjay-poddar/Dev-Geeks#652] [Q&A] [answered]
Is this still acceptable for Hacktoberfest(i mean in 2022)?

**Comment by pranjay-poddar** [comment]
@Aanya9693 yes

**Comment by Thre4dripper** [comment]
Absolutely

**Comment by pranjay-poddar** [comment, marked as answer]
> Is this still acceptable for Hacktoberfest(i mean in 2022)?

Please refer to readme, everything is mentioned.

**Comment by HuyenSuong20222** [comment]
@Thre4dripper heiss noob
